### PR TITLE
removed urllib3 restriction

### PR DESCRIPTION
**Pull request by jonas-w**
## Before this PR

We wanted to release new version of foundry-dev-tools to conda-forge but
it failed because of the urllib3 restrictions you set. (<2.0.0)

## After this PR

The urllib3 version will be restricted by the requests library
but not by your restriction, which seems to be unneccessary.
This should allow us to release new foundry-dev-tools version on conda-forge.

## Possible downsides?

-

## Are Docs needed?

-

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/palantir-oauth-client](https://github.com/palantir/palantir-oauth-client), @jonas-w! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: pyproject.toml [hunk 1]
```diff
@@ -14,14 +17,11 @@ python = "^3.8"
 oauthlib = "^3.2.2"
 requests = "^2.28.2"
 requests-oauthlib = "^1.3.1"
-urllib3 = "^1.26.14"
+# urllib3 = "^1.26.14"
```
Comment (tom-s-powell) [inline pyproject.toml]: Could you remove this line rather than comment it out?

Comment (jonas-w) [reply pyproject.toml]: Sure!

**Review by tom-s-powell — APPROVED** [review]

**Comment by tom-s-powell** [comment]
Mind clicking the checkbox to generate the changelog?

**Comment by jonas-w** [comment]
> Mind clicking the checkbox to generate the changelog?

I already tried it yesterday, I can't click any of these checkboxes.

**Pull request merged**